### ISSUE TITLE
Indicate what we're creating a classification for

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#315]
```
Ok, adding new datafile.
Please enter a name: rep_test1.rep                                                          
Ok, please provide classification for new entry: 
   1) Search an existing classification
   2) Add a new classification
   .) Cancel import                                                                         
 >                     
```

In the above case, I'm pretty sure I'm providing the classification for the datafile.  But on other occasions it is less clear, particularly around platforms & sensors.

So, in the instruction above, we should replace `entry` with the data-type in question.  It's `datafile` in this instance.  I guess this will just mean the addition of a parameter to the get_privacy method.

**Comment by BarisSari** [comment]
Okay, Ian. I'm including it to the this PR: #307 .

**Comment by BarisSari** [comment]
Fixed in #307
